### PR TITLE
[PR5] DEV-70 add ExcelAdapter

### PR DESCRIPTION
**Pull request by jmaralo**
The ExcelAdapter downloads the excel and parses its contents.

This is the same as the previous ExcelAdapter, the only change is the name of the sheets and the Compile method, which takes various objects that can be built from the boards and puts them together

### CODE REVIEW

File: src/excel_adapter/excel_adapter.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	"path/filepath"
+
+	"github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/internals"
+	internalModels "github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/internals/models"
+	"github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/models"
+	"github.com/xuri/excelize/v2"
+)
+
+const GlobalSheet = "Info"
+const AddressesTable = "addresses"
```
Comment (smorsua) [inline src/excel_adapter/excel_adapter.go:15]: Put each address in the corresponding board sheet

**Review by smorsua — APPROVED** [review]
LGTM!

File: src/excel_adapter/internals/excel_reader.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+	return models.Sheet{
+		Tables: tables,
+	}
+}
+
+func findTables(cols [][]string) map[string][4]int {
+	tables := make(map[string][4]int)
+	for i, col := range cols {
+		for j, cell := range col {
```
Comment (lipezaballa) [inline src/excel_adapter/internals/excel_reader.go:67]: may be separate the second for in another function?

File: src/excel_adapter/internals/excel_reader.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	}
+	return -1
+}
+
+func findTableHeight(cols [][]string, firstCol int, firstRow int, width int) int {
+	maxHeight := 0
+	height := 0
+	for _, col := range cols[firstCol : width+firstCol] {
+		for j, cell := range col[firstRow+2:] { //firstRox+2: first data row
+			if cell == "" {
```
Comment (lipezaballa) [inline src/excel_adapter/internals/excel_reader.go:102]: same, another function, maybe not necessary, but more manageable

**Review by lipezaballa — APPROVED** [review]

**Pull request merged**